lib/usb_serial.dart

```diff
@@ -335,12 +341,12 @@ class UsbDevice {
     if (!(other is UsbDevice)) {
       return false;
     }
-    return _Equality.deepEq(_props, other._props);
```

Comment:
Instead of removing _props lets put deviceName in props on line 331 instead of the current list of items. 

`List<Object?> get _props => [deviceName];`

Reply:
why is it better? it will still compare all the props (and that's a non-sense imo)

Reply:
It allows us to add more properties in the future in case we missed something important here.

Reply:
Not sure i understand. I can see it's used for equals() and hashCode() only and for sure we don't need anything else. You can always revert and introduce back _props if/when you need.

I'd vote for cleaning-up (my 2nd commit in the MR), but feel free to do what you think is better.